app/addons/documents/mango/mango.api.js

```diff
@@ -104,9 +128,10 @@ export const mergeFetchParams = (queryCode, fetchParams) => {
   };
 };
 
-export const mangoQueryDocs = (databaseName, queryCode, fetchParams) => {
-  const url = FauxtonAPI.urls('mango', 'query-server', encodeURIComponent(databaseName));
+export const mangoQuery = (databaseName, queryCode, fetchParams) => {
+  const url = FauxtonAPI.urls('mango', 'query-server', databaseName);
```

Comment:
Can yo uadd the encodeURIComponent back here. Otherwise this doesn't work with any database with `/` in it